ca_qc_saint_jerome/people.py

```diff
@@ -1,40 +1,35 @@
-# coding: utf-8
 from utils import CanadianScraper, CanadianPerson as Person
 
-COUNCIL_PAGE = 'http://www.vsj.ca/fr/membres-du-conseil.aspx'
+import re
+
+COUNCIL_PAGE = 'https://www.vsj.ca/conseil-municipal-et-comite-executif/membres-du-conseil-municipal'
 
 
 class SaintJeromePersonScraper(CanadianScraper):
     def scrape(self):
-        page = self.lxmlize(COUNCIL_PAGE)
-
-        councillors = [tr for tr in page.xpath('//table//tr[1]') if len(tr) == 2][:-1]
-        assert len(councillors), 'No councillors found'
-        for i, councillor in enumerate(councillors):
-            texts = [text.strip() for text in councillor.xpath('.//text()[normalize-space()]') if text.strip()]
+        page = self.lxmlize(COUNCIL_PAGE, encoding='utf-8')
+        councillors = page.xpath('//div[contains(@class," inner_member")]')
 
-            if 'vacant' in texts[1]:
-                continue
+        for councillor in councillors:
+            name = councillor.xpath('.//h2/text()')[0]
+            assert len(councillors), 'No councillors found'
```

Comment:
Please keep the assert outside the for-loop.

Reply:
done